Makefile

```diff
@@ -37,7 +39,7 @@ so:
 		"SunOS")  make sun-cxform.so \
 				"CFLAGS=-fPIC -shared -I$(IDL_EXT_UNIX)" ;;\
 		"Darwin") make mac-cxform.so \
-				"CFLAGS=-fPIC -arch x86_64 -arch i386 -arch ppc -arch ppc64 -I$(IDL_EXT_UNIX)" ;;\
+				"CFLAGS=-fPIC -arch x86_64 -I$(IDL_EXT_UNIX)" ;;\

```

Comment:
Should this be IDL_EXT_MAC ? If not, perhaps lines 24 & 29 should be removed since they aren't referenced?